internal/applicationsnapshot/report.go

```diff
@@ -35,6 +35,7 @@ type Component struct {
 	Violations []conftestOutput.Result  `json:"violations"`
 	Warnings   []conftestOutput.Result  `json:"warnings"`
 	Success    bool                     `json:"success"`
+	Passed     int                      `json:"passed"`
```

Comment:
`Successes` or `Passed`? I'm leaning towards `Passed`

Reply:
Maybe `successes` because that's also a noun like `warnings` and `violations`?

Reply:
This is a good point. Conftest uses Passed, but I can't find a good reason.

Reply:
One note: We want to (in the future) actually list details about the successes. So maybe reserve `successes` for that purpose and use something like SuccessCount for this.

Reply:
`Passed` isn't so bad though, idk.

Reply:
Oh! I thought this was about adding the list of passing rules, not just the count. +1 to Simon's suggestion.